### PR TITLE
fix: avoiding duplicate make manifest dep and minor fix on pipeline yaml

### PR DESCRIPTION
**Pull request by danielherrerohernando**
### Type:
* [ ] cicd: helm, docker & cicd adjustments.
* [ ] feature: new capabilities.
* [x] fix: bug, hotfix, etc.
* [ ] refactor: enhacements.
* [ ] style: changes in styles.
* [ ] other: docs, tests.

## What’s the focus of this PR
This PR just updates one minor error in template's azure-pipeline file related to Docker registry connection.
Also removes duplication in template's package.json for the make-manifest package

## How to review this PR
Just check those files in the template

Before submitting this PR, I made sure:

- [x] The code builds clean without any errors or warnings
- [x] I'm using our guidelines

### CODE REVIEW

**Review by MatteoDiPaolo — APPROVED** [review]
LGTM

**Review by kevinccbsg — APPROVED** [review]

File: generators/app/templates/root/_package.json [hunk 1]
```diff
@@ -37,7 +37,6 @@
     "debug": "^2.2.0",
     "express-jsdoc-swagger": "^1.0.5",
     "hogan.js": "^3.0.2",
-    "make-manifest": "^1.0.1",
```
Comment (Betisman) [inline generators/app/templates/root/_package.json:40]: Why not removing the devDependency instead?

Comment (danielherrerohernando) [reply generators/app/templates/root/_package.json:40]: @Betisman As far as I know "make-manifest" is not part of the actual code executed at runtime, it's just a script to be run prior to the release in order to generate kind of a "info file" so it should be placed in devDependencies along with other "helper" code not actually needed at runtime (tests, docs...)

Comment (Betisman) [reply generators/app/templates/root/_package.json:40]: Ok

**Pull request merged**